templates/components/search/criteria_filter.html.twig

```diff
@@ -60,7 +61,7 @@
               <p class="empty-title">{{ __("No filter found") }}</p>
               <p class="empty-subtitle text-muted">{{ __("There is no filter defined yet for this item.") }}</p>
               <div class="empty-action">
-                <a id="enable_filter_action" class="btn btn-primary"><i class="ti ti-plus me-2"></i>{{ __("Create a filter") }}</a>
+                <button id="enable_filter_action" class="btn btn-primary"><i class="ti ti-plus me-2"></i>{{ __("Create a filter") }}</button>
```

Comment:
```suggestion
                <button id="enable_filter_action" type="button" class="btn btn-primary"><i class="ti ti-plus me-2"></i>{{ __("Create a filter") }}</button>
```
Probably doesn't matter as long as this is not in a form.

Reply:
👍